components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/ClaimMetadataManagementAdminService.java

```diff
@@ -238,6 +239,11 @@ public void addExternalClaim(ExternalClaimDTO externalClaim) throws ClaimMetadat
 
             IdentityClaimManagementServiceDataHolder.getInstance().getClaimManagementService()
                     .addExternalClaim(claim, tenantDomain);
+        } catch (ClaimMetadataClientException e) {
+            if (log.isDebugEnabled()) {
+                log.debug(e.getMessage());
```

Comment:
stack trace is printed with debug log. it should be log.debug(e.getMessage(), e);